pandas/core/groupby/__init__.py

```diff
@@ -0,0 +1,20 @@
+import warnings
+
+# TODO: Remove after 0.XX.x
+warnings.warn("'pandas.core.groupby' has moved. Use "
+              "'pandas.core.groupby.groupby' instead", FutureWarning,
+              stacklevel=2)
+
+from pandas.core.groupby.groupby import Grouper  # noqa: F401
+from pandas.core.groupby.groupby import groupby  # noqa: F401
+from pandas.core.groupby.groupby import BinGrouper  # noqa: F401
```

Comment:
only put public functions here. If something is needed internally then change the reference to it.